examples/test_aws_full_governed_ws/init.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  backend "s3" {
+    bucket = "databricks-terraform-blueprints2"
+    key    = "test_aws_full_governed_ws.tfstate"
+    region = "us-east-1"
+  }
+  required_providers {
+    databricks = {
+      source  = "databricks/databricks"
+      version = "~>1.6.5"
```

Comment:
I've updated all the `required_providers` block to the latest version, let's just make sure the tests still pass (and upgrade the `.lock.hcl` with `terraform init --upgrade`

Reply:
Thanks @nkvuong , all updates made. Please check and let me know if anything else is pending.